frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { t } from "ttag";
+import { Link } from "react-router";
+import {
+  Description,
+  Header,
+  Label,
+  StyledCard,
+  StyledIcon,
+  MoreDetails,
+} from "./EmbeddingOption.styled";
+
+interface EmbeddingOptionProps {
+  setting: {
+    embedName: string;
+    embedDescription: string;
+    embedType: "standalone" | "full-app";
+  };
+}
+
+export default function EmbeddingOption({ setting }: EmbeddingOptionProps) {
```

Comment:
This component render the following 2 cards
![image](https://user-images.githubusercontent.com/1937582/176174293-e943106d-7356-40c9-8144-a5fdc4b4b4f8.png)